packages/data-point/lib/transform-expression/factory.js

```diff
@@ -82,7 +83,10 @@ function create (source = []) {
   const tokens = parse(source)
 
   const transformBase = new TransformExpression()
-  transformBase.reducers = tokens.map(reducerFactory.create)
+
+  transformBase.reducers = tokens.map(token => {
```

Comment:
I think it's worth documenting there (inline comment) that you have recursiveness happening... good thing to warn other devs

Reply:
thanks for adding the comment